app/src/main/java/com/kickstarter/ui/activities/SettingsActivity.kt

```diff
@@ -3,62 +3,76 @@ package com.kickstarter.ui.activities
-import com.kickstarter.libs.BaseActivity
 import com.kickstarter.libs.Build
 import com.kickstarter.libs.KSString
 import com.kickstarter.libs.Logout
-import com.kickstarter.libs.qualifiers.RequiresActivityViewModel
 import com.kickstarter.libs.rx.transformers.Transformers
 import com.kickstarter.libs.transformations.CircleTransformation
 import com.kickstarter.libs.utils.ApplicationUtils
 import com.kickstarter.libs.utils.ViewUtils
+import com.kickstarter.libs.utils.extensions.addToDisposable
+import com.kickstarter.libs.utils.extensions.getEnvironment
 import com.kickstarter.viewmodels.SettingsViewModel
 import com.squareup.picasso.Picasso
-import rx.android.schedulers.AndroidSchedulers
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
 
-@RequiresActivityViewModel(SettingsViewModel.ViewModel::class)
-class SettingsActivity : BaseActivity<SettingsViewModel.ViewModel>() {
+class SettingsActivity : AppCompatActivity() {
     private lateinit var build: Build
     private lateinit var ksString: KSString
     private lateinit var logout: Logout
     private var logoutConfirmationDialog: AlertDialog? = null
     private lateinit var binding: SettingsLayoutBinding
+    private lateinit var disposables: CompositeDisposable
+
+    private lateinit var viewModelFactory: SettingsViewModel.Factory
+    private val viewModel: SettingsViewModel.SettingsViewModel by viewModels {
+        viewModelFactory
+    }
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = SettingsLayoutBinding.inflate(layoutInflater)
+        this.getEnvironment()?.let { env ->
+            viewModelFactory = SettingsViewModel.Factory(env)
+        }
+        disposables = CompositeDisposable()
 
         setContentView(binding.root)
 
         if (BuildConfig.DEBUG) {
             binding.editProfileRow.visibility = View.VISIBLE
         }
 
-        this.build = requireNotNull(environment().build())
-        this.ksString = requireNotNull(environment().ksString())
-        this.logout = requireNotNull(environment().logout())
+        this.build = requireNotNull(getEnvironment()?.build())
+        this.ksString = requireNotNull(getEnvironment()?.ksString())
+        this.logout = requireNotNull(getEnvironment()?.logout())
 
         binding.versionNameTextView.text = ksString.format(
             getString(R.string.profile_settings_version_number),
             "version_number", this.build.versionName()
         )
 
         this.viewModel.outputs.avatarImageViewUrl()
-            .compose(bindToLifecycle())
-            .compose(Transformers.observeForUI())
-            .subscribe { url -> Picasso.get().load(url).transform(CircleTransformation()).into(binding.profilePictureImageView) }
+            .compose(Transformers.observeForUIV2())
+            .subscribe { url ->
+                Picasso.get().load(url).transform(CircleTransformation())
```

Comment:
fyi there is an Image Extension that does exactly the same on `com.kickstarter.ui.extensions.ImageViewExtKt#loadCircleImage`